src/main/java/seedu/address/model/person/Attribute.java

```diff
@@ -20,11 +20,19 @@ public T getValue() {
     }
 
     /**
-     * Determine if a specified value is a match with the value stored in this
+     * Gets the displayable value stored in this attribute.
+     *
+     * @return Displayable value stored in this attribute.
+     */
+    public T getDisplayableValue() {
+        return this.value;
+    }
+
```

Comment:
Is there a reason can't just be a method in the `Phone` class? It does the same thing as `getValue()`

Reply:
The introduction of this method is for extensibility in future. It is not definite that all attributes are expected to only be stored as a single `String` value. Speaking of which, it might make more sense for `getDisplayableValue` to return a `String` instead.

Adding it only to the `Phone` class will make it confusing to developers when actually displaying the values as developers have to now navigate between multiple methods (`getValue()` and `getDisplayableValue`) to display attributes.

While it can be argued whether this logic should be implemented in the `toString` method, one reason to add `getDisplayableValue` is in the clarity of it's name, indicating its purpose, whereas `toString` may be better suited for more generic use cases such as containing the formatting logic that makes it easier for test cases.